main.py

```diff
@@ -191,31 +191,63 @@ def to_payload(self):
         return r
 
 
+class Point(object):
+    def __init__(self, row: int, col: int) -> None:
+        if not isinstance(row, int):
+            raise ValueError('row')
+        if not isinstance(col, int):
+            raise ValueError('col')
```

Comment:
I find the noise-to-utility ratio of this error low - it does not save the user from anything and chances are high that a developer will find the invalid range quickly.

Reply:
You know you handle responses from an external program not under your control?